TestOIDC Failure in Vault 1.9.0

Vault 1.9.0 introduced the following error in test_oidc.py. Creating an issue to track the resolution of this failure.

```
FAILED tests/integration_tests/api/auth_methods/test_oidc.py::TestOIDC::test_oidc_authorization_url_request_0_success - hvac.exceptions.InvalidRequest: cannot find key "oidc-test-key", on post https://localhost:8200/v1/identity/oidc/role/hvac-oidc-test
```

--- a/tests/integration_tests/api/auth_methods/test_oidc.py
+++ b/tests/integration_tests/api/auth_methods/test_oidc.py
@@ -197,6 +197,9 @@ class TestOIDC(HvacIntegrationTestCase, TestCase):
             )
         id_token_role_name = "hvac-oidc-test"
         key_name = "oidc-test-key"
+        create_named_key_response = self.client.secrets.identity.create_named_key(
+            name=key_name,
+        )
         create_or_update_role_response = (
             self.client.secrets.identity.create_or_update_role(
                 name=id_token_role_name,

